docs/src/references/mappers/c8y-mapper.md

```diff
@@ -209,6 +209,42 @@ and hence there is no mapping done for main device registration messages.
 Inventory data updates for the main device are handled differently.
 :::
 
+## Auto Registration of Entities
+
+Before any data messages from an entity can be processed, the entity has to be registered first.
+The entities can be registered either explicitly, or implicitly,
+depending on the auto-registration mode set with `c8y.entity_store.auto_register`.
+
+To register an entity explicitly, an entity registration message has to be sent.
+
+But, if the auto-registration is enabled, sending any data message on a topic following the default topic scheme (`te/device/<device-id>/service/<service-id>/...`),
+results in the auto-registration of that target entity.
+	
+For example, sending a measurement message to `te/device/child1///m/temperature`,
+will result in the auto-registration of the entity with topic id: `device/child1//` and the auto-generated external id: `<main-device-id>:device:child1`, derived from the topic id.
+Similarly, a measurement message on `te/device/child1/service/my-service/m/temperature`,
+results in the auto-registration of both the device entity: `device/child1//` and the service entity: `device/child1/service/my-service` with their respective auto-generated external IDs, in that order.
+
+The auto-registration can be disabled/enabled using the `tedge config` command as follows:
+
+```sh
+sudo tedge config set c8y.entity_store.auto_register false
+```
+:::note
+When the auto registration is disabled, and if the device is not already registered.
+Then the c8y-mapper will not forward the incoming telemetry/health-status messages to the c8y cloud.
+But, logs/publishes an error message on the `te/errors` topic telling the entity is not registered.
```

Comment:
```suggestion
When the auto registration is disabled, and if the device is not explicitly registered,
then the c8y-mapper will just ignore all the data messages received from that device,
logging that error message on the `te/errors` topic telling the entity is not registered.
```
Reasons for rewording: 
1. The first sentence was ending abruptly
2. The messages are not limited to telemetry and health-status messages.